guides/v2.1/cloud/deploy/reduce-downtime.md

```diff
@@ -0,0 +1,34 @@
+---
+group: cloud-guide
+title: Reduce downtime
+functional_areas:
+  - Cloud
+  - Deploy
+  - SCD
+---
+
+{{site.data.var.ece}} runs Magento in [_maintenance_ mode]({{page.baseurl}}/config-guide/bootstrap/magento-modes.html#maintenance-mode) during the deploy phase, which takes your site offline until the deployment is complete. The length of time your Production site is in maintenance mode depends on the size of the site, the amount of changes applied during the deployment, and the configuration for static content deployment.
+
+During the deployment process, all connections queue for up to 5 minutes preserving any active sessions and pending actions, such as adding to cart or checkout. After deployment, the queue is released and connections continue without interruption. To use this _connection hold_ to your advantage and reduce your downtime to zero, you must configure your project to employ the most efficient deploy strategy possible: SCD on build with HTML minification.
+
+Use the following steps to help reduce the amount of time it takes your store to deploy an update to Production:
```

Comment:
It is not exactly minimizing downtime, but rather avoiding it. There is a window of time they need to take advantage of so that deployment time doesn't surpass the connection hold and they experience downtime.